explorer/interpreter/value_transform.h

```diff
@@ -179,13 +180,13 @@ class ValueTransform : public TransformBase<Derived> {
 
   // Preserve vtable during transformation.
   auto operator()(Nonnull<const VTable* const> vtable)
```

Comment:
We don't generally `const`-qualify parameters even if they're not changed in the function body, so for consistency I'd suggest changing this one too:
```suggestion
  auto operator()(Nonnull<const VTable*> vtable)
```